pycbc/workflow/minifollowups.py

```diff
@@ -642,6 +644,15 @@ def make_qscan_plot(workflow, ifo, trig_time, out_dir, injection_file=None,
             if trig_time in seg:
                 data_seg = seg
                 break
+            elif trig_time == -1.0:
```

Comment:
I think I'm in favour of the `if trig_time == -1.0: DO_NOT_MAKE_A_PLOT` option ... But will approve this code either way, and it can be changed later.